### DISCUSSION TITLE
GRAPE method without using any backend solver

### DISCUSSION THREAD

**Discussion by STaherpour** [JuliaQuantumControl/QuantumControl.jl#20] [Q&A] [answered]
Hi 
Is it possible to use GRAPE method for optimizing without using any backend solver such as LBFGSB in QuantumControl.jl package? For example in [simple_state_to_state](https://juliaquantumcontrol.github.io/GRAPE.jl/stable/examples/simple_state_to_state/)

**Comment by goerz** [comment, marked as answer]
I don’t understand. How would you optimize without a solver?

**Reply by STaherpour** [reply]
Sorry for replying late. I find out my problem. Thanks a lot.
